app/design/adminhtml/Magento/backend/web/css/source/components/_modals_extend.less

```diff
@@ -146,13 +146,13 @@
     }
 
     .action-close {
-        padding: @modal-popup__padding;
+        padding: 1rem;
 
         &:active,
         &:focus {
             background: transparent;
-            padding-right: @modal-popup__padding + (@modal-action-close__font-size - @modal-action-close__active__font-size) / 2;
-            padding-top: @modal-popup__padding + (@modal-action-close__font-size - @modal-action-close__active__font-size) / 2;
+            padding-right: 1rem;
```

Comment:
Hi @cedarvinda. Thanks for collaboration. Please use `less` variables to declare padding size.

Reply:
Hi @VladimirZaets Thank for updating for this .. I will use less variable.

Reply:
Hello @VladimirZaets Now added less variables to declare padding size.